### ISSUE TITLE
Fraud in the AI industry, Vaporware aka "Proof of concept, or GTFO"

### ISSUE THREAD

**Issue by chrisbward** [HumanAIGC/AnimateAnyone#27]
Too much of this going on.

It's all too easy to fake results and attain funding - this needs to stop!

https://github.com/magic-research/magic-animate for a working alternative

**Comment by Randy-H0** [comment]
This isn't faked. You can still see artifacting, unnatural things going on and theyre all perfectly in sync. Please take a look at the results

**Comment by Collin-Budrick** [comment]
Too many people, such as yourself, who are impatient and shallow-minded. The developers said they have every intention to release the code for our testing pleasure. If the code is not shown as advertised, then it will reflect badly on their company, and we will all move on to another project.

**Quit pretending you paid for a counterfeit product and are demanding a refund.**

**Comment by chrisbward** [comment]
This indeed does reflect badly.

**Comment by Collin-Budrick** [comment]
If they released it and it works as demoed in the video... everyone will return here and have a good laugh. (myself included)

**Comment by visionmaster2** [comment]
> Too much of this going on.
> 
> It's all too easy to fake results and attain funding - this needs to stop!
> 
> https://github.com/magic-research/magic-animate for a working alternative

Before insulting people, their work, and making a fool of yourself, you should research the individuals who created this document.

**Comment by chrisbward** [comment]
> If they released it and it works as demoed in the video... everyone will return here and have a good laugh. (myself included)

And I will apologise, sincerely.

**Comment by chrisbward** [comment]
I'm happy to be proven wrong - but do please understand that this is a systemic issue within the growing space.

Whether it's vulnerability exploitation, blockchain projects or even LK99 claims - there has to be pushback from the community for proof of concept. This cannot be the new norm.

**Comment by Randy-H0** [comment]
> I'm happy to be proven wrong - but do please understand that this is a systemic issue within the growing space.
> 
> Whether it's vulnerability exploitation, blockchain projects or even LK99 claims - there has to be pushback from the community for proof of concept. This cannot be the new norm.

Literally look at the results. They are not perfect in any way, you can see obvious arrifacting when you look at these results. Please do your research before going "oh. Fake" when you see the first few results, not even looking for flaws.

**Comment by formulake** [comment]
Guilty till proven innocent hardly seems like a responsible approach.

**Comment by cepera-ang** [comment]
@chrisbward what exactly are examples of fraud in academic community researching AI topics? And do you really think that Alibaba researches publish this paper to attract funding?

**Comment by chrisbward** [comment]
It's not a difficult concept to understand. We shall wait and see.

**Comment by wweevv-johndpope** [comment]
https://github.com/guoqincode/AnimateAnyone-unofficial

**Comment by DrakenZA** [comment]
> This isn't faked. You can still see artifacting, unnatural things going on and theyre all perfectly in sync. Please take a look at the results

I dont think he means like, its real ppl dancing and they faking it. I think he means, its pretty easy to do these sorts of things with already available tech, and make out like you have created something 'new' in order to get funding etc

Github turning into a place for teams to advertise, and at any point now, its gonna turn super fake, super fast.

**Comment by chrisbward** [comment]
> > This isn't faked. You can still see artifacting, unnatural things going on and theyre all perfectly in sync. Please take a look at the results
> 
> I dont think he means like, its real ppl dancing and they faking it. I think he means, its pretty easy to do these sorts of things with already available tech, and make out like you have created something 'new' in order to get funding etc
> 
> Github turning into a place for teams to advertise, and at any point now, its gonna turn super fake, super fast.

Precisely this. I am posting the exact same sentiment on all projects that follow this pattern. It's not to target this one per-se.

It's a larger issue with where the community and industry is headed!

**Comment by ShawnFumo** [comment]
Though honestly, I'm not sure AA is the best example of that anyway. It isn't like there was a big marketing campaign that I know of. I think there was a few people that watch papers coming out who profiled it first and then all the AI influencers on twitter decided to run with it. I don't think Alibaba even made a tweet on it themselves.

**Comment by cepera-ang** [comment]
@chrisbward So, that means you're harassing multiple research teams with baseless accusations of misconduct? I really don't understand why people choose to come and be angry at a team that haven't invited anyone, doesn't sell anything, don't need public funding, just sharing their research. No one advertising anything on github, the repo isn't even for users, it's for fellow researchers.

Expectations are becoming more unrealistic every day. Not only it should be product quality from the day one, also completely free, open-source and there still be accusations of fraud or at least of being part of the trend "that there soon be only fraud". 

Chillout and let the people work.

**Comment by sroussey** [comment]
> @chrisbward So, that means you're harassing multiple research teams with baseless accusations of misconduct? I really don't understand why people choose to come and be angry at a team that haven't invited anyone, doesn't sell anything, don't need public funding, just sharing their research. No one advertising anything on github, the repo isn't even for users, it's for fellow researchers.

This repo hit popular press. So random people came to see, not knowing that the standard in these things is to publish first, and then put up code (if code is delivered at all) a week to a couple of months later. The good stuff is in the math in the paper, of course. But script kiddies want to get something to run that they don't understand.

This project should not have opened issues until the code dropped. Might be a new thing going forward. @HumanAIGC should definitely close this one.

**Comment by cepera-ang** [comment]
@sroussey yeah, maybe. Quick look at Chris Linkedin reveals that he is some kind of consultant in generative content area, so he probably should know better then demand freebies from industry researchers while basically building business on their work and accusing "industry" of fraud.

**Comment by chrisbward** [comment]
> @sroussey yeah, maybe. Quick look at Chris Linkedin reveals that he is some kind of consultant in generative content area, so he probably should know better then demand freebies from industry researchers while basically building business on their work and accusing "industry" of fraud.

Actually a dev of 23 years, I have a couple of side projects, I contribute a lot to open-source projects.
I'm not demanding any "freebies".

I have some experience in exploit development and this shit doesn't fly in those circles.

I'm closing the ticket, but I wouldn't hold my breath.

**Comment by Randy-H0** [comment]
Stop opening this

**Comment by chrisbward** [comment]
> Stop opening this

no!

**Comment by chrisbward** [comment]
![Waiting-Skeleton](https://github.com/HumanAIGC/AnimateAnyone/assets/888374/a7703a3f-e7e7-470d-84dd-259a7303f270)

**Comment by CRYPTO-GENIK** [comment]
These could be generated by an i stagram filter with face tracking, which seems more likely.

Alibaba doesn't inspire confidence by showing examples with no explanation

**Comment by Randy-H0** [comment]
> > These could be generated by an i stagram filter with face tracking, which seems more likely.
> > Alibaba doesn't inspire confidence by showing examples with no explanation
> 
> How do you mean no explanation. You know there is a whole detailed research paper devoted to this right? https://arxiv.org/abs/2311.17117
> 
> And people have already implemented open source versions based on the paper. Not as good as the original, but still showing that it works in general.

Dont worry, these people don't actually know what a research paper is. No wonder they didn't find one

**Comment by chrisbward** [comment]
6 months, no code - clowns

**Comment by Collin-Budrick** [comment]
> 6 months, no code - clowns

You're 100% correct. You called, and I mistakenly doubted you.

**Comment by chrisbward** [comment]
https://www.youtube.com/watch?v=NIaC-t2ao6E

**Comment by chrisbward** [comment]
Guys, NGL GTA6 is going to drop before this 😭😭😭

**Comment by chrisbward** [comment]
![Image](https://github.com/user-attachments/assets/a91c0f33-e3e1-48db-a5db-fc7e41b417ee)
